spec/lib/govuk_elements_form_builder/form_builder_spec.rb

```diff
@@ -48,6 +48,19 @@ def size(method, size)
       ]
     end
 
+    it 'supports attributes defined as a string' do
+      output = builder.send method, 'name', class: 'custom-class'
```

Comment:
Is there a reason you've used `'name'` rather than `:name` as in the test directly above? It makes it look like two things are varying in this test when it's actually only the `class` (if I'm reading it right?).